share/spice/twitch_streams/twitch_streams.js

```diff
@@ -0,0 +1,59 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_twitch_streams = function(api_result){
+
+        if (!api_result || !api_result.streams || api_result.streams.length === 0 ||api_result.error) {
+            return Spice.failed('twitch_streams');
+        }
+        
+        var streams = [];
+        if (!($.isArray(api_result.streams))) {
+            streams = [api_result.streams];
+        } else {
+            streams = api_result.streams;
+        }  
+        
+        //Get query, split into an array on space, append query to the end for moreat
+        var script = $('[src*="/js/spice/twitch_streams/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/twitch_streams\/([^\/]+)/)[1],
+            decodedQuery = decodeURIComponent(query);
+        var moreAt = decodedQuery.replace(/ /g,"+");
```

Comment:
This can be shortened: 
```javascript
        var script = $('[src*="/js/spice/twitch_streams/"]')[0],
            source = $(script).attr("src"),
            query = decodeURIComponent(source.match(/twitch_streams\/([^\/]+)/)[1]),
            moreAt = query.replace(/ /g,"+");
```